app/services/analytics_events.rb

```diff
@@ -2952,7 +2953,7 @@ def idv_usps_auth_token_refresh_job_started(**extra)
   # @param [DateTime] enqueued_at When was this letter enqueued
   # @param [Integer] which_letter Sorted by enqueue time, which letter had this code
   # @param [Integer] letter_count How many letters did the user enqueue for this profile
-  # @param [Integer] attempts Number of attempts to enter a correct code
+  # @param [Integer] submit_attempts Number of attempts to enter a correct code
```

Comment:
A little unsure if this should be "code entry attempts" or something
instead...but the definition of this attribute seems to be from the
same areas I have renamed `submit_attempts`, so that's why I chose that.